pubspec.yaml

```
@@ -12,7 +12,7 @@ dependencies:
   checked_yaml: ^1.0.0
   # Verified that no new features since 1.0.0 are used - be careful!
   json_annotation: '>=1.0.0 <5.0.0'
-  pub_semver: ^1.3.2
+  pub_semver: ^2.0.0
```

How about a range for now?

```suggestion
  pub_semver: ">=1.3.0 <3.0.0"
```

If you feel we should be using this as a new lower bound, please change the commit message from "allow" to "require"

---

yup. oops. I plan to publish ASAP